cap/modules/deposit/links.py

```diff
@@ -13,7 +14,16 @@ def links_factory(pid):
 
     # Update 'html' key to point to the frontend app
     # links['html'] = links['html'].replace('/app/', '/')
+    def _url(name, **kwargs):
```

Comment:
hopefully, you can remove this when https://github.com/inveniosoftware/invenio-deposit/pull/163 is merged.